### PR TITLE
fix: update Slider after dir change

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-2004

+Style fix for prompt - moving margin to gapMatch, regression of https://github.com/oat-sa/extension-tao-itemqti/pull/1953

**Steps to Reproduce**
Create an item with LTR language (e.g. English).
Add a Slider interaction.
Change the language of the item to an RTL language (e.g. Arabic).

**Actual Result**
The lower bound stays on the left side.
The upper bound stays on the right side.

**Expected Result**
The lower bound is on the right side.
The upper bound is on the left side.

### CODE REVIEW

**Review by Silvia-loza — APPROVED** [review]
All looks good & works as expected. 
Great the modification of the comment it really helps understand how it works better.  :bulb:
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not` master`
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Review by marpesia — CHANGES_REQUESTED** [review]
It is working well when I change the Item language, but not if I access to auth an item with Arabic already set.
Can you take a look at this, please?

image

File: views/js/qtiCreator/widgets/item/Widget.js [hunk 1]
```diff
@@ -107,6 +107,7 @@ define([
                 if (item.bdy.attr('dir') === 'rtl') {
                     // dir='rtl' should be set to itemBody
                     $itemBody.attr('dir', 'rtl');
+                    $itemBody.trigger('item-dir-changed');
```
Comment (bziondik) [inline views/js/qtiCreator/widgets/item/Widget.js:110]: @marpesia here the fix for loaded item

**Review by marpesia — APPROVED** [review]
LGTM

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [ ] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**